uco-master/uco.ttl

```diff
@@ -41,3 +41,27 @@
 		;
 	.
 
+<http://unifiedcyberontology.org/observable#CyberAction>
+	rdfs:subClassOf <http://unifiedcyberontology.org/action#Action> ;
+	.
+
+<http://unifiedcyberontology.org/observable#CyberItem>
+	rdfs:subClassOf <http://unifiedcyberontology.org/core#Item> ;
```

Comment:
This should be added directly to the CyberItem definition in observable.ttl not in master.ttl.
I have a separate commit to address this fix.

Reply:
I'll import core.ttl into observable.ttl and make CyberItem a subclass of Observable. I'll make the CyberItem a subclass of Item in observable.ttl. I will remove the subclass assertions from uco.ttl.